### PR TITLE
Allow to change multipart options from getFileStream

### PR DESCRIPTION
**Pull request by sm2017**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by dead-horse** [comment]
can you add a test case for this feature?

**Comment by sm2017** [comment]
Can you please add test yourself
I don't know what test case can be added
I just pass a parameter

File: app/extend/context.js [hunk 1]
```diff
@@ -65,12 +65,14 @@ module.exports = {
...
    *  - {Boolean} options.requireFile - required file submit, default is true
+   *  - {Object} options.multipart - override default multipart configurations
    * @return {ReadStream} stream
    * @since 1.0.0
    */
   async getFileStream(options) {
     options = options || {};
-    const parts = this.multipart({ autoFields: true });
+    options.multipart = options.multipart || {};
+    const parts = this.multipart(Object.assign({ autoFields: true, options.multipart }));
```
Comment (fengmk2) [inline app/extend/context.js]: > options are passed to busboy. The only additional option is autoFields.

`getFileStream()` only work on `autoFields = true`, so you can't change it.

Comment (sm2017) [reply app/extend/context.js]: So we can do the following

```js
    const parts = this.multipart(Object.assign({ }, options.multipart ,{autoFields: true}));

```

Comment (sm2017) [reply app/extend/context.js]: @fengmk2 what about
```js
    const parts = this.multipart(Object.assign({ }, options.multipart ,{autoFields: true}));
 ```

Comment (sm2017) [reply app/extend/context.js]: @fengmk2 I opend https://github.com/eggjs/egg-multipart/pull/25

**Pull request closed without merging**